src/screens/profile/edit-profile/edit-profile.tsx

```diff
@@ -63,9 +65,17 @@ const EditProfile: React.FC<ProfileStackScreenProps<'EditProfile'>> = ({ navigat
           </FormControl>
         </VStack>
 
-        <Button onClick={() => formik.handleSubmit()} isLoading={isUpdateAccountLoading}>
-          Save Changes
-        </Button>
+        <VStack space={4} mt={8}>
+          <Divider
+            orientation={DividerOrientation.Horizontal}
+            thickness={1}
+            style={{ marginVertical: 16 }}
```

Comment:
This is not expected. Caller should not be able to pass `style`, it easily lead to component abstracting the design